### PR TITLE
Bug/#35 Fix confirm password filed in RegisterComponent

### PR DESCRIPTION
**Pull request by FFreestyler**

### CODE REVIEW

**Comment by FFreestyler** [comment]
[Fix confirm password filed in RegisterComponent](https://trello.com/c/Zbeh2DUD/35-fix-confirm-password-filed-in-registercomponent)

File: src/components/auth/RegisterComponent.tsx [hunk 1]
```diff
@@ -60,7 +60,11 @@ function RegisterComponent({ changeComponentType }: any) {
               <AuthFormField fieldName="username" required autoFocus />
               <AuthFormField fieldName="email" required />
               <AuthFormField fieldName="password" required />
```
Comment (IHiFiveI) [inline src/components/auth/RegisterComponent.tsx]: Why not add the `type="password"` for regular password field?

Comment (StepanovRoma) [reply src/components/auth/RegisterComponent.tsx]: I don't know why, but this field is already like a password

**Review by MangriMen — CHANGES_REQUESTED** [review]
Need remove "type" from AuthFormFiled to make type clearer to see

**Review by MangriMen — APPROVED** [review]

**Review by StepanovRoma — APPROVED** [review]

**Pull request merged**